traitlets/traitlets.py

```diff
@@ -2262,6 +2264,8 @@ class CBool(Bool):
 
     def validate(self, obj, value):
         try:
+            if isinstance(value, str):
```

Comment:
You should use `six.string_types` here, and add some byte strings to the `_good_values` test.

Reply:
done